src/SprykerEco/Zed/Unzer/Persistence/UnzerEntityManagerInterface.php

```diff
@@ -40,9 +40,32 @@ public function savePaymentUnzerTransactionEntity(
     ): PaymentUnzerTransactionTransfer;
 
     /**
-     * @param \Generated\Shared\Transfer\MerchantUnzerParticipantTransfer $merchantUnzerParticipantTransfer
+     * @param \Generated\Shared\Transfer\UnzerCredentialsTransfer $unzerCredentialsTransfer
      *
-     * @return \Generated\Shared\Transfer\MerchantUnzerParticipantTransfer
+     * @return \Generated\Shared\Transfer\UnzerCredentialsTransfer
      */
-    public function saveMerchantUnzerParticipantEntity(MerchantUnzerParticipantTransfer $merchantUnzerParticipantTransfer): MerchantUnzerParticipantTransfer;
+    public function createUnzerCredentials(UnzerCredentialsTransfer $unzerCredentialsTransfer): UnzerCredentialsTransfer;
+
+    /**
+     * @param array $idStores
+     * @param int $idUnzerCredentials
+     *
+     * @return void
+     */
+    public function addUnzerCredentialsStoreRelationsForStores(array $idStores, int $idUnzerCredentials): void;
+
+    /**
+     * @param array $idStores
+     * @param int $idUnzerCredentials
+     *
+     * @return void
+     */
+    public function removeUnzerCredentialsStoreRelationsForStores(array $idStores, int $idUnzerCredentials): void;
```

Comment:
Imho we should be consistent with the naming and follow create, update, delete.

addUnzerCredentialsStoreRelationsForStores  => createUnzerCredentialsStoreRelationsForStores
removeUnzerCredentialsStoreRelationsForStores => deleteUnzerCredentialsStoreRelationsForStores